package.json

```diff
@@ -35,49 +35,49 @@
   },
   "devDependencies": {
     "@gravitational/build": "workspace:*",
-    "@storybook/addon-actions": "^8.3.4",
-    "@storybook/addon-controls": "^8.3.4",
-    "@storybook/addon-toolbars": "^8.3.4",
-    "@storybook/components": "^8.3.4",
-    "@storybook/preview-api": "^8.3.4",
-    "@storybook/react": "^8.3.4",
-    "@storybook/react-vite": "^8.3.4",
-    "@storybook/test-runner": "^0.19.1",
-    "@testing-library/jest-dom": "^6.5.0",
+    "@storybook/addon-actions": "^8.4.6",
+    "@storybook/addon-controls": "^8.4.6",
+    "@storybook/addon-toolbars": "^8.4.6",
+    "@storybook/components": "^8.4.6",
+    "@storybook/preview-api": "^8.4.6",
+    "@storybook/react": "^8.4.6",
+    "@storybook/react-vite": "^8.4.6",
+    "@storybook/test-runner": "^0.20.0",
+    "@testing-library/jest-dom": "^6.6.3",
     "@testing-library/react": "^16.0.0",
     "@testing-library/user-event": "^14.5.2",
     "@types/history": "^4.7.11",
-    "@types/jest": "^29.5.13",
-    "@types/node": "^20.16.10",
-    "@types/react": "^18.3.10",
-    "@types/react-dom": "^18.3.0",
+    "@types/jest": "^29.5.14",
+    "@types/node": "^20.17.9",
```

Comment:
I suspect that this broke types in teleport.e, I'm yet to figure out how to fix this.

https://github.com/gravitational/teleport.e/blob/b7ee0144cfbc3e1ec28a6f6bda4efc72b86122a5/web/teleport/src/SAMLIdPLogin/SAMLIdPLogin.tsx#L39

> e/web/teleport/src/SAMLIdPLogin/SAMLIdPLogin.tsx:39:54 - error TS2345: Argument of type `ArrayBufferLike` is not assignable to parameter of type `ArrayBuffer`.
  Type `SharedArrayBuffer` is missing the following properties from type `ArrayBuffer`: resizable, resize, detached, transfer, transferToFixedLength
```
39         const urlSafeMfaResponse = bufferToBase64url(mfaResponseBytes.buffer);
```

Reply:
Oh, maybe it was actually a TS update that broke this?

https://devblogs.microsoft.com/typescript/announcing-typescript-5-7-rc/#support-for---target-es2024-and---lib-es2024

Reply:
I wonder if a `as ArrayBuffer` is too simple a hack

Reply:
I'd say that would work if it was a temporary workaround for some 3rd-party dep issue. But based on that TypeScript post, it looks like we need to figure out a way to correctly use those APIs.

They mentioned that if you have those type errors, you should look into upgrading `@types/node`, but our version is already the newest one for v20.